### DISCUSSION TITLE
SEB flickers when running

### DISCUSSION THREAD

**Discussion by ChiaraR17** [SafeExamBrowser/seb-win-refactoring#333] [Q&A] [answered]
Hallo,
I have a problem running SEB properly on my Windows 11.
I have installed the latest version of SEB 3.3.1 and set up a configuration file using the moodle plug-in. Now when trying to open SEB via the "config.seb"-file. The screen flickers. (see video)

Same happens on Windows 10.
Maybe you can help me solve the issue

https://user-images.githubusercontent.com/68544308/149892220-0f5e9cd3-d048-49ed-adf7-946c45b807f1.mp4

?

**Comment by janiber** [comment]
Hello,

I have the same issiue. Out of 20 students one seems to have this switching between Moodle login and SEB running screen. After running SEB, student is locked in SEB.
Windows 10, SEB 3.3.1

On another computer i have SEB 3.3.1 and all works fine..
Any ideas what is causing this?


https://user-images.githubusercontent.com/97946404/149911840-997ce450-6ca0-4e75-ab4e-a1959e2bb60d.mp4

1

**Comment by dbuechel** [comment]
It appears that there is an issue with the initialization of the kiosk mode. Could both of you please provide the log files of the affected session(s)? They can be found under `%LocalAppData%\SafeExamBrowser\Logs`.

**Reply by janiber** [reply]
Thank you for your reply, here are the log files from the session.
[Logs.zip](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7888437/Logs.zip).

**Reply by ChiaraR17** [reply]
[2022-01-18_09h16m47s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7888448/2022-01-18_09h16m47s_Runtime.log)

Thank you for your reply, please find the log here.

**Reply by dbuechel** [reply]
Yes, as I have suspected, there is something on both of your systems which interferes with the kiosk mode of SEB, namely reactivating the Windows desktop once SEB has created its own, custom desktop:

`2022-01-17 22:56:39.778 [18] - WARNING: [DesktopMonitor] Detected desktop switch to 'Default' [3852], trying to reactivate 'SafeExamBrowser' [3692]...`

Are you using any anti-virus or other special security software, or do you use any custom Windows modifications, cleaner utilities or alike? If yes, would you mind sharing which ones exactly?

**Reply by janiber** [reply]
Well in my case, the student said he only has windows defender and what Windows 10 has in default, no other antivirus program.

**Reply by ChiaraR17** [reply]
As well, in my case, I´m just using windows defender and just updated to windows 11, no other modifications.

Is there any possibility we could see, if some modifications/setting is trying to reactivate the windows desktop?

**Reply by dbuechel** [reply]
If possible, could you please make a video which shows the entire procedure from the beginning, i.e. including startup and initialization of SEB? It does appear like there is something interfering with SEB as said above, but we can't really do much without a way to reproduce the issue on our side as well.

**Reply by ChiaraR17** [reply]
Please find attached a video when startin SEB, the used configuration file and the according log files, maybe this could help
[Log_Configuration_Video.zip](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7890082/Log_Configuration_Video.zip)
.

**Reply by janiber** [reply]
In attached file you can find our video and log files.
[mov2.zip](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7895474/mov2.zip)
.

**Reply by dbuechel** [reply]
Thanks @ChiaraR17 and @janiber. Please try to close all applications before starting SEB, especially Webex and other meeting or screen sharing software and then try to start SEB again.

**Comment by juliussaputra** [comment, marked as answer]
Hello guys

This just happened to me as well out of sudden. But after trying some stuff I found the culprit.
For me the problem was Webex app or process. If I close the app before starting Safe Exam Browser, the flicker is gone!
![image](https://user-images.githubusercontent.com/17081818/150167838-d7dfd0b8-51d7-4287-a9f2-3bead6629c52.png)

This is only happened on a specific version tho, in my case Webex Version:  42.1.0.21190.
My other colleague with version 41.12.0.20899 doesn't have the flicker problem.

Hope it helps!

Regards
Julius

**Reply by dbuechel** [reply]
Thanks for your input @juliussaputra. I just have watched the videos of @ChiaraR17 and @janiber and have noted that there were a lot of applications running (especially for @ChiaraR17), among them also Webex. According to the quick research I have done it might very well be that Webex attempts to reactivate the Windows desktop as it also has a screen sharing function.

**Reply by ChiaraR17** [reply]
Thank you a lot, @dbuechel and @juliussaputra I closed webex and SEB was now working properly!

**Reply by janiber** [reply]
I can also confirm this. We are also using latest version (42.1.0.21190) of Webex and it appears that is what was causing the problem. After closing Webex SEB worked properly.

Thank you!

**Reply by janiber** [reply]
We are using Webex to monitor the students (using video and sound) during exams.
We would appreciate if this problem can be resolved.

**Reply by dbuechel** [reply]
> We are using Webex to monitor the students (using video and sound) during exams. We would appreciate if this problem can be resolved.

I fear that this would need to be solved on Webex's side, as they shouldn't switch desktops but rather always capture the currently active desktop the user is on. We could (and probably will) add Webex to the default list of prohibited applications in order to prevent this malfunction by default.

In the meantime, if you'd like to use Webex then you could try to configure it as a permitted application: https://safeexambrowser.org/windows/win_usermanual_en.html#ApplicationsPane. You'd however need to make sure that Webex first is terminated when starting SEB and then start it again from within the running SEB session. If you're out of luck however, Webex still will try to activate the default desktop in which case you'd definitely need to try to report the issue to them.

**Reply by janiber** [reply]
We have notified Webex about this issue and we are hoping it can be resolved. Perhaps rather than adding Webex to list of prohibited applications it would be better to just inform users that SEB is not compatible with recent (42.1.0.21190) version of Webex. 

For now we have asked our students to (1) have older version of Webex or (2) to close Webex on their computers when usin SEB and connect to Webex via mobile phone. untill this is resolved. 

Again thank @dbuechel, @juliussaputra and @ChiaraR17

**Reply by danschlet** [reply]
We will definitely add Webex to the default list of prohibited applications, because SEB is a secure kiosk browser, which by default always is supposed to lock down computers during an exam. Most people wouldn't want students cheating by having a Webex session running along SEB, wouldn't they?
Therefore if you're using Webex for proctoring purposes, you will need to deactivate the default prohibited process for Webex and might have to add it as a permitted process in your SEB settings.

**Comment by matejjost** [comment]
I contacted Cisco's Webex development engineers to resolve this issue, and we were successful. They sent me the new (development - beta) versions (42.2.0.213** and 42.3.0.2****) and it seems that this bug is fixed. I hope that these updated version will be available for everyone soon.

**Reply by dbuechel** [reply]
That's excellent news, thanks for the update!
